lib/Ed25519VerificationKey2020.js

```diff
@@ -30,11 +38,29 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
   constructor(options = {}) {
     super(options);
     this.type = SUITE_ID;
-    this.publicKeyMultibase = options.publicKeyMultibase;
-    if(!this.publicKeyMultibase) {
+    const {publicKeyMultibase, privateKeyMultibase} = options;
+
+    if(!publicKeyMultibase) {
       throw new TypeError('The "publicKeyMultibase" property is required.');
     }
-    this.privateKeyMultibase = options.privateKeyMultibase;
+
+    if(!publicKeyMultibase || !_isValidKeyHeader(
+      publicKeyMultibase, MULTICODEC_ED25519_PUB_HEADER)) {
+      throw new Error(
+        '`publicKeyMultibase` has invalid header bytes: ' +
+        `"${publicKeyMultibase}".`);
+    }
+
+    if(privateKeyMultibase && !_isValidKeyHeader(
+      privateKeyMultibase, MULTICODEC_ED25519_PRIV_HEADER)) {
+      throw new Error('privateKeyMultibase` has invalid header bytes.');
```

Comment:
Use double quotes and fix missing starting quote:
```suggestion
      throw new Error('"privateKeyMultibase" has invalid header bytes.');
```